### DISCUSSION TITLE
MOES BRT-100-TRV intermittently failing to respond. Batteries out is the only way to fix it.

### DISCUSSION THREAD

**Discussion by nick4275** [Koenkk/zigbee2mqtt#19863] [Devices]
There are a bunch of us over on the HA Community Forum with this issue. No one seems to be able to pin down what might be occurring. Looking for some assistance if anyone can help.
So the units pair up fine and work fine generally.
Occasionally and for no reason I can find, they fail to respond to POST from Z2M. Although they will respond to GET.
Once it has entered this state it virtually never recovers (although it always shows online). Dropping out the batteries and re-booting the unit will always fix it for a while.
For example when a unit is not behaving; say a TRV is set to 10°C and I send a command to raise this to 20°C it will appear to accept the command without error. If I then query the device a few seconds (or minutes) later the device will return the original value of 10°C. No matter how many times thereafter I send commands to the device it will not change. 
TIA

**Comment by slowmotion85** [comment]
Hello, i'm having the same problem with my TRVs. When the unit fails to repsond, it appears on z2m map as connected to the router/coordinator but with an LQI of 0.
I'm not sure if it's linked or not, but i noticed that activating new devices' inclusion highly increases the probability of TRV failures. I haven't got any proof of this yet, so i need to test some more.
I'm using the sonoff CC2652P zigbee coordinator and a bunch of routers, the TRV that hangs the most is the one in the same room as the coordinator (again, i don't know if it's just a coincidence or not).
It's a very hard issue to debug, since we can't know at which exact momenet the TRV fails to respond, and be able to link that to a specific action. I had everything working for two weeks without a problem, and when i tried to include a new device on my network three TRVs failed to respond the same day.
I'll let you know if i manage to get solid conclusions of this behavior.

**Comment by slowmotion85** [comment]
So as i went to see what was happening on my zigbee network after my comment, and this is what i saw:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/48499660/89b08b14-a943-4ed0-bcea-dc55ec93fede)

A message saying that all my TRVs didn't respond at the same time; that was strange.
I went to z2m logs, and at the exact same time:
`
info  2024-01-11 09:05:33: Zigbee: allowing new devices to join.
info  2024-01-11 09:05:33: MQTT publish: topic 'zigbee-rdc/bridge/response/permit_join', payload '{"data":{"value":true},"status":"ok"}'
`
I'm starting to think this is more than just a coïncidence.
And by the way, my configurations file says "permit_join: false", so i don't know why it activates this function by itself.
I have another coordinator in the basement, and it seems that it also activated the permit_join by itself at 9:05:13 this morning.

Do you have any idea about what can cause this behavior?
Regards.

**Comment by Ra72xx** [comment]
Many of the users experiencing bug https://github.com/Koenkk/zigbee2mqtt/issues/19747 have most of the connection issues with thermostats (Aqara, however). Maybe a similar problem?
